aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageDownloadFileOperation.java

```diff
@@ -173,11 +173,7 @@ public void onStateChanged(int transferId, TransferState state) {
                     onSuccess.accept(StorageDownloadFileResult.fromFile(file));
                     return;
                 case FAILED:
-                    onError.accept(new StorageException(
-                            "Storage download operation was interrupted.",
-                            "Please verify that you have a stable internet connection."
-                    ));
-                    return;
+                    // no-op;
```

Comment:
*Note:* `onError.accept` is already invoked in an error scenario on line 195 below

https://github.com/aws-amplify/amplify-android/blob/b7c442e92c722c2b853d2efc6c79143859a4ad62/aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageDownloadFileOperation.java#L191-L200